### PR TITLE
Add missing visible symbols in let expression with no variable declaration

### LINKED ISSUE #34144: Missing visible symbols in let expression
**Description:**
$Title

**Steps to reproduce:**
Screenshot 2021-12-03 at 10 27 23

**Related Issues (optional):**
https://github.com/ballerina-platform/ballerina-lang/issues/33236

### PR DESCRIPTION
**Pull request by gayalkuruppu**
Fixes #34144 

Screenshot 2021-12-10 at 13 05 39

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Review by nadeeshaan — DISMISSED** [review]
LGTM

**Comment by mohanvive** [comment]
@gayalkuruppu please resolve the conflict.

File: language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/type_test_expression_ctx_config8.json [hunk 1]
```diff
@@ -444,15 +444,15 @@
     {
       "label": "module1:TestMap2",
       "kind": "TypeParameter",
-      "detail": "Map",
+      "detail": "Typereference",
```
Comment (IMS94) [inline language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/type_test_expression_ctx_config8.json:447]: @malinthar IMO, the detail here should be `Map`. Do you have an idea why this changed here?

**Comment by gayalkuruppu** [comment]
Fixed in master, hence closing the PR

**Pull request closed without merging**